padertorch/train/hooks.py

```diff
@@ -358,6 +374,9 @@ def pre_step(self, trainer: 'pt.Trainer'):
     def close(self, trainer):
         self._save_latest_checkpoint(trainer)
 
+    def set_last(self, iteration, epoch):
+        pass
```

Comment:
Why do you disable `set_last` for `CheckpointHook`?

Reply:
May be problematic with Backoff.

Reply:
This was to make CheckpointHook rewrite the checkpoint after backoff with updated remaining_backoffs. But you are right, this was not very clean. Now the CheckpointHook checks whether it is a backoff or resume and sets last accordingly such that a checkpoint is only rewritten after backoff.